app/components/shared/inputs/index.ts

```diff
@@ -62,6 +62,7 @@ export interface IListOption<TValue> {
 export interface IMediaGalleryMetadata extends IInputMetadata {
   fileName: string;
```

Comment:
Why we need `fileName` here? We already have `value` property

Reply:
`fileName` is a display value whereas `value` is the href, but i suppose we could move the filename parsing into the input component that probably makes more sense

Reply:
Yeah, we probably just need a getter here.

Reply:
To be clear

Reply:
To be clear, the `metadata` property intended to store only values that are not going to be changed when you change the `value` property.